src/styles/prices.scss

```diff
@@ -0,0 +1,26 @@
+@import 'variables';
+
+.prices {
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto 16px;
+  align-items: center;
+  width: 166px;
+  height: 18px;
```

Comment:
The fixed width and height for the `.prices` block can make the layout non-responsive and inflexible. If the content changes or the font size is adjusted, these dimensions may no longer be appropriate. Consider using padding and allowing the content to define the size of the block, or use min-width and min-height to ensure flexibility. This is a code quality issue as it affects the scalability and maintainability of the styles.